src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -143,12 +143,14 @@
          content-item]
 
         :else
-        [react/view {:position        :absolute
-                     :left            56
-                     :right           56
-                     :height          styles/toolbar-height
-                     :justify-content :center
-                     :align-items     :center}
+        [react/view
+         (merge {:position        :absolute
```

Comment:
so this is a global change for all toolbars?

Reply:
yes.. now that you point out.. this can be a problem.. I'll fix it.. I think in `profile.user.views` we can use negative margin (-16) instead

Reply:
did go with the alternative implementation. This file is untouched now.